library/core/src/cell.rs

```diff
@@ -753,6 +753,22 @@ impl Display for BorrowMutError {
     }
 }
 
+// This ensures the panicking code is outlined from `borrow_mut` for `RefCell`.
+#[inline(never)]
+#[track_caller]
+#[cold]
+fn panic_already_borrowed(err: BorrowMutError) -> ! {
+    panic!("already borrowed: {:?}", err)
```

Comment:
Now this will print on release "already borrowed: BorrowMutError" instead of "already borrowed", which is useless.

Reply:
That's done to match the existing output. `BorrowMutError` may also contain the location of the original borrower.

Reply:
https://play.rust-lang.org/?version=nightly&mode=release&edition=2021&gist=4db78fa88be92de81e74933887a7a140
Oh, my bad, with this pr and previously that code outputs "already borrowed: BorrowMutError".

Sad to not to have test coverage.

Can this be simple strings like "already borrowed" instead? If user wants more info from error he will use try_borrow/try_borrow_mut instead of borrow/borrow_mut, that fn should be as slim as possible. And location available only via `debug_refcell` which require rebuild std anyway?

Reply:
It would make sense to me to use simpler strings with `debug_refcell` disabled.